### PR TITLE
Add Ansible role to deploy

### PR DESCRIPTION
**Pull request by Amygos**
The ansible role will install the `fpp` instance and create the configuration for the brands, and the relative Traefik configurations.

### CODE REVIEW

File: deploy/ansible/roles/fpp/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            apple_team_id: 4tf7of1hqb
+            apple_key_id: cit75cjk43
+            apple_topic: com.example.MyApp
+            apple_environment: production
+            instance_token: 1cde32b5-fdc0-464d-86ee-6a91dab8fe27
+```
+
+## License
+
+MIT
```
Comment (gsanchietti) [inline deploy/ansible/roles/fpp/README.md]: Can we move the role to GPLv3 like the whole project?

File: deploy/ansible/roles/fpp/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    - role: fpp
+      vars:
+        brands:
+          example:
+            http_port: 8081
+            google_credentials_file: credentials.json
+            apple_credentials_file: credentials.p8
+            apple_team_id: 4tf7of1hqb
+            apple_key_id: cit75cjk43
+            apple_topic: com.example.MyApp
```
Comment (gsanchietti) [inline deploy/ansible/roles/fpp/README.md]:
```suggestion
            apple_topic: com.example.MyApp.voip
```

File: deploy/dynamic.yml [hunk 3]
```diff
@@ -1,72 +0,0 @@
-http:
```
Comment (gsanchietti) [inline deploy/dynamic.yml]: It's ok to remove these files, but should also update the deploy section of the readme: https://github.com/nethesis/fpp/blob/master/README.md#build-and-deploy

**Review by gsanchietti — COMMENTED** [review]
PR is good, just a couple of minor fixes

File: deploy/dynamic.yml (hunk 3 above)
Comment (Amygos) [reply deploy/dynamic.yml]: ok, I will update the main README

File: README.md [hunk 4]
```diff
@@ -206,6 +206,9 @@ badger info --read-only --dir . --show-keys
 
 ## Build and deploy
 
+Fpp can be deployed manually or using the [Ansible role](deploy/ansible/roles/fpp/README.md).
+
+### Manually setup
```
Comment (gsanchietti) [inline README.md]:
```suggestion
### Manual setup
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]
This part is not working anymore: https://github.com/nethesis/fpp/pull/5/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L241-L251

I'd totally remove the manual deploy and leave only the build one.

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**